packages/block-editor/src/components/inserter/test/reusable-blocks-tab.native.js

```diff
@@ -50,15 +48,15 @@ describe( 'ReusableBlocksTab component', () => {
 		const reusableBlockItems = items.filter(
 			( { category } ) => category === 'reusable'
 		);
-		const component = shallow(
+		const component = render(
 			<ReusableBlocksTab
 				rootClientId={ 0 }
 				onSelect={ jest.fn() }
-				listProps={ {} }
+				listProps={ { contentContainerStyle: {} } }
 			/>
 		);
-		expect( component.find( BlockTypesList ).prop( 'items' ) ).toEqual(
-			reusableBlockItems
-		);
+		reusableBlockItems.forEach( ( { title } ) => {
+			component.getByText( title );
```

Comment:
This test lacks of `expect` statements so I'm wondering if we could update with something like:
```suggestion
			expect( component.getByText( title ) ).toBeTruthy();
```